packages/schematics/angular/ng-new/index_spec.ts

```diff
@@ -103,4 +103,22 @@ describe('Ng New Schematic', () => {
     const { cli } = JSON.parse(tree.readContent('/bar/angular.json'));
     expect(cli.packageManager).toBe('npm');
   });
+
+  it('should experimentalZoneless is set to true and add provideExperimentalZonelessChangeDetection in app.config', async () => {
```

Comment:
`should add 'provideExperimentalZonelessChangeDetection()' when 'experimentalZoneless' is true`